.github/workflows/lib-injection.yml

```diff
@@ -26,66 +26,56 @@ jobs:
-        echo "Using image name '$DOCKER_IMAGE_NAME_WITH_TAG'"
-        echo "image_name=$DOCKER_IMAGE_NAME_WITH_TAG" >> $GITHUB_OUTPUT
+        mkdir lib-injection/dd-trace-py/
+        rsync -a . lib-injection/dd-trace-py/ --exclude lib-injection
+     
+    - name: lib-injection-tags
+      id: lib-injection-tags
+      uses: DataDog/system-tests/lib-injection/docker-tags@robertomonteromiguel/lib_injection_integration_v2
+      with:
+        init-image-name: 'dd-lib-python-init'
+        main-branch-name: 'robertomonteromiguel-lib_injection_system_tests_integration'
+
     - name: Login to Docker
       run: docker login -u publisher -p ${{ secrets.GITHUB_TOKEN }} ghcr.io
+
     - name: Docker Build
       uses: docker/build-push-action@v3
       with:
         push: true
-        tags: ${{ steps.set_names.outputs.image_name }}
-        platforms: "linux/amd64,linux/arm64/v8"
+        tags: ${{ steps.lib-injection-tags.outputs.tag-names }}
+        platforms: 'linux/amd64,linux/arm64/v8'
         build-args: "DDTRACE_PYTHON_VERSION=${{ steps.get_version.outputs.library_version }}"
         context: ./lib-injection
 
-  build-and-publish-sample-app-images:
+  injection-tests:
+    needs:
+      - build-and-publish-init-image
     runs-on: ubuntu-latest
+    permissions:
+      contents: read
+      packages: write
     strategy:
       matrix:
-        image-name: [
-          "dd-lib-python-init-test-django",
-          "dd-lib-python-init-test-django-gunicorn",
-          "dd-lib-python-init-test-django-uvicorn",
-        ]
-    steps:
-      - uses: actions/checkout@v3
-
-      - name: Set up QEMU
-        uses: docker/setup-qemu-action@v2
-
-      - name: Set up Docker Buildx
-        id: buildx
-        uses: docker/setup-buildx-action@v2
-
-      - name: Available platforms
-        run: echo ${{ steps.buildx.outputs.platforms }}
-
-      - name: Set Docker Image Tag
-        id: set_names
-        run: |
-          DOCKER_IMAGE_TAG=${{ github.sha }}
-          DOCKER_IMAGE_NAME=${{ matrix.image-name }}
-          DOCKER_IMAGE_NAME=$(echo ghcr.io/${GITHUB_REPOSITORY}/${DOCKER_IMAGE_NAME} | tr '[:upper:]' '[:lower:]')
-          DOCKER_IMAGE_NAME_WITH_TAG=$(echo ${DOCKER_IMAGE_NAME}:${DOCKER_IMAGE_TAG} | tr '[:upper:]' '[:lower:]')
-          echo "Using image name '$DOCKER_IMAGE_NAME_WITH_TAG'"
-          echo "image_name=$DOCKER_IMAGE_NAME_WITH_TAG" >> $GITHUB_OUTPUT
-      - name: Login to Docker
-        run: docker login -u publisher -p ${{ secrets.GITHUB_TOKEN }} ghcr.io
-      - name: Docker Build
-        uses: docker/build-push-action@v3
+        lib-injection-connection: [ 'network','uds']
+        lib-injection-use-admission-controller: ['', 'use-admission-controller']
+        weblog-variant: [ 'dd-lib-python-init-test-django','dd-lib-python-init-test-django-gunicorn','dd-lib-python-init-test-django-uvicorn']
+      fail-fast: false
+    env:
+      TEST_LIBRARY: python
+      WEBLOG_VARIANT: ${{ matrix.weblog-variant }}
```

Comment:
```suggestion
      VARIANT: ${{ matrix.variant }}
```

Reply:
I would like to maintain this name, only because in system-tests we called as "weblog". In the future, I would like to use existing "weblog" variants in system-tests, instead to have custom sample apps for lib-injection